test/rpc/test_rpc_misc.py

```diff
@@ -121,3 +150,25 @@ def test_call_with_invalid_params(params):
     # could be any legal method, just passing something to get params to fail
     ex = rpc_call(method="starknet_getClass", params=params)
     assert ex["error"] == {"code": -32602, "message": "Invalid params"}
+
+
+@pytest.mark.usefixtures("run_devnet_in_background")
+@pytest.mark.parametrize(
+    "run_devnet_in_background, input_data, expected_data",
+    GET_EVENTS_TEST_DATA,
+    indirect=True,
+)
+def test_get_events(input_data, expected_data):
+    """
+    Test RPC get_events.
+    """
+    deploy_info = deploy(EVENTS_CONTRACT_PATH)
+    for i in range(0, 2):
```

Comment:
The more usual way is to do `for i in range(2)`

Reply:
fixed